### PR TITLE
Documentation: Remove section on how to obtain anonymized ID (no longer required for getting student info in ORA)

### PR DESCRIPTION
**Pull request by catong**
DOC-1641 -- documentation for TNL-836
This PR removes the mentions of and steps for obtaining the anonymized student ID and replaces it with "enter the student's username".
@explorerleslie @muzaffaryousaf @symbolist can you please review for technical accuracy and workflow?
@mhoeber @srpearce @lamagnifica can any one of you please do an editorial pass?
Thanks!

### CODE REVIEW

**Comment by lamagnifica** [comment]
@catong the Research guide is affected as well...

**Comment by muzaffaryousaf** [comment]
:+1:

**Comment by catong** [comment]
@lamagnifica Thanks - I'll catch up with you today on this. I'm not familiar enough with the Research guide -- could use some more specific pointers.

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst [hunk 1]
```diff
@@ -51,50 +51,18 @@ In the following example, you can see the student's response. The response recei
 
 For an example that shows a student's response with more assessments, see :ref:`Access Student Information`.
 
-Accessing information about a specific student has two steps:
+For details about accessing information for a specific student, see :ref:`Access Student Information`.
```
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst]: could the xrefs on lines 52 and 54 be consolidated into a single one?

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst [hunk 2]
```diff
@@ -51,50 +51,18 @@ In the following example, you can see the student's response. The response recei
...
 
 =======================================
-Access the Student's Information
+Access a Specific Student's Information
 =======================================
 
 #. In the LMS, go to the peer assessment assignment that you want to see.
 #. Scroll to the bottom of the problem, and then click the black **Course Staff Information** banner.
-#. Scroll down to the **Get Student Info** box, paste the student's course-specific anonymized user ID in the box, and then click **Submit**.
+#. Scroll down to the **Get Student Info** box, enter the student's username in the box, and then click **Submit**.
```
Comment (lamagnifica) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst]: there's an awful lot of scrolling -- needed as a direction in all three steps? also, this last step has three different directions in it -- break into 2-3 steps?

**Comment by lamagnifica** [comment]
some suggestions, use what works. +1 on this file. 
For the research guide, the question to ask is, are there any changes to the openassessmentblock.get_peer_submission, openassessmentblock.peer_assess, and openassessmentblock.self_assess events, or to the user_id_map SQL table? they all store anonymized student IDs. (I don't think there will be, but it wouldn't hurt to make sure...)
Also, is there any change or effect on this section: http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/running_course/course_student.html#access-anonymized-student-ids ?

**Comment by catong** [comment]
@lamagnifica Thanks. No effect on the other mentions of the anonymized student IDs elsewhere in the doc. They still exist and can be obtained in the same way. The change in this story is that you no longer need to use an anonymized student ID to see the ORA student info. You use a student username instead. No changes to anonymized student ID use elsewhere.

**Comment by catong** [comment]
@symbolist @muzaffaryousaf 
Can you please verify per @lamagnifica's comment above, whether there has been any impact to openassessmentblock.get_peer_submission, openassessmentblock.peer_assess, and openassessmentblock.self_assess events, or to the user_id_map SQL table?

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 1 above)
Comment (catong) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst]: Ha! Thanks!

File: en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst (hunk 2 above)
Comment (catong) [inline en_us/shared/subsections/open_response_assessments/Access_ORA_Info.rst]: Yes

**Comment by muzaffaryousaf** [comment]
@catong @lamagnifica this story was only about to get the student_info based on his username. There would be no effect on these methods. Thanks.

**Comment by explorerleslie** [comment]
:+1:

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/134

**Comment by catong** [comment]
@jhendersonedx FYI

**Pull request merged**